src/content-feature.js

```diff
@@ -222,4 +243,14 @@ export default class ContentFeature {
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     update () {
     }
+
+    /**
+     * Register a string flag that will be added to page breakage reports
+     * @param {string} flag
+     */
+    addDebugFlag (flag = 'fired') {
+        this.debugMessaging?.notify('addDebugFlag', {
```

Comment:
This method **needs** to memoise and fire only once, to prevent swamping the native messaging with debugging if the site does something like:

```
const pixels = ctx.getImageData(0, 0, canvas.width, canvas.height);
for (let i = 0; i<imageData.data.length; i++) {
  if (navigator.deviceMemory) {
  ...
  }
}
```